### DISCUSSION TITLE
How to change language recognition for RapidVideoOCR

### DISCUSSION THREAD

**Discussion by Mamacito473** [SWHL/RapidVideOCR#40] [Q&A] [answered]
Currently it's trying to OCR in chinese, I want it to OCR in french 

![0_00_55_000__0_00_57_207_0000000001920108019201080](https://github.com/SWHL/RapidVideOCR/assets/38729166/e14d1a1c-1323-472a-9bd8-cc952257ffe0)

as you can see there is chinese & french hardsubbed in the video ,

which variables do I need to change in which config files to enable rapid video OCR to OCR only in french ? 

thank you guys

**Comment by SWHL** [comment]
When I update the project document, there will be detailed steps in the new document.
Please stay tuned.

**Comment by SWHL** [comment, marked as answer]
I write a [blog](https://swhl.github.io/RapidVideOCR/docs/blog/-rec_other_lan/) for changing language with RapidVideOCR in Chinese.

Hope it helps you.
